MvvmCross/Binding/Droid/Target/MvxImageViewDrawableTargetBinding.cs

```diff
@@ -45,7 +46,15 @@ protected override void SetValueImpl(object target, object value)
                 imageView.SetImageDrawable(null);
             else
             {
-                imageView.SetImageResource(intValue);
+				var appContext = AndroidGlobals.ApplicationContext;
+				Drawable drawable;
+				if (Build.VERSION.SdkInt >= BuildVersionCodes.Lollipop)
+					drawable = appContext.Resources.GetDrawable(intValue, appContext.Theme);
```

Comment:
Shouldn't this use the `ImageView`'s context?  The app context could have a different theme than the activity's.